common/kubernetes/configs.go

```diff
@@ -120,9 +120,6 @@ func GetKubeClientConfig(uri *url.URL) (*kube_client.Config, error) {
 	if len(kubeConfig.Host) == 0 {
 		return nil, fmt.Errorf("invalid kubernetes master url specified")
 	}
-	if len(kubeConfig.GroupVersion.Version) == 0 {
```

Comment:
I'd say you should set kubeConfig.GroupVersion at line 111, that's the only path that could end up with a nil GroupVersion and cause the panic. I don't know why this problem is exposed now.